pyhms/sprout/sprout_candidates.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
+
+import numpy as np
+
+from ..core.individual import Individual
+
+
+@dataclass
+class DemeFeatures:
+    """Class containing the numerical information calculated from the deme state before sprouting."""
+
+    "NBC_mean_distance: Mean distance between individuals and their nearest better neighbor."
+    NBC_mean_distance: float | np.float64 = None
```

Comment:
we don't use uppercase for HMS - why do you use it for nbc?

Reply:
it should be 
```
NBC_mean_distance: float | np.float64 | None = None
```

Reply:
What's the purpose of this string?
```
"NBC_mean_distance: Mean distance between individuals and their nearest better neighbor."
```
I would prefer to add this comment to the docstring